Mathlib/Combinatorics/SimpleGraph/Clique.lean

```diff
@@ -299,41 +299,28 @@ theorem cliqueFree_completeMultipartiteGraph {ι : Type*} [Fintype ι] (V : ι 
   exact absurd he hn
 
 /-- Clique-freeness is preserved by `replaceVertex`. -/
-theorem cliqueFree_of_replaceVertex_cliqueFree [DecidableEq α] (s t : α) (h : G.CliqueFree n) :
+protected theorem CliqueFree.replaceVertex [DecidableEq α] (h : G.CliqueFree n) (s t : α) :
```

Comment:
Note about dot notation: there's no need to put `G.CliqueFree` first.

I think it's nicer in this new order though.